File: 단계별로 풀어보기/기본 수학 1/부녀회장이 될테야.c
#include <stdio.h>
main(){
    int t;
	int n, k;
	int i, j, l;
	int people;
	scanf("%d", &t);
	for(i = 0; i < t; i++){
		scanf("%d", &n);
		scanf("%d", &k);
		
		int apart[n + 1][14];
		for(j = 0; j < 14; j++)
			apart[0][j] = j + 1;
		
		for(l = 0; l <= n; l++)
			apart[l][0] = 1;
		
		for(l = 1; l <= n; l++){
			for(j = 1; j < 14; j++){
				apart[l][j] = apart[l][j - 1] + apart[l - 1][j];
				
			}
		}
		
		printf("%d\n", apart[n][k - 1]);
	}
   }

File: 단계별로 풀어보기/기본 수학 1/부녀회장이 될테야.py
t = int(input())
apart = []
for i in range(t):
    n = int(input())
    k = int(input())
    apart = [[0 for _ in range(14)] for _ in range(n + 1)]
    
    for j in range(14):
        apart[0][j] = j + 1

    for l in range(n + 1):
        apart[l][0] = 1
    for l in range(1, n + 1):
        for j in range(1, 14):
            apart[l][j] = apart[l][j - 1] + apart[l - 1][j]


    

    print(apart[n][k - 1])
